test_runner/fixtures/zenith_fixtures.py

```diff
@@ -128,15 +129,29 @@ class Postgres:
     def __init__(self, zenith_cli):
         self.zenith_cli = zenith_cli
         self.running = False
+        self.host = 'localhost'
+        self.port = 55432
         # path to conf is <repo_dir>/pgdatadirs/pg1/postgresql.conf
 
-    def create_start(self):
+    def create_start(self, config_lines=[]):
         """ create the pg data directory, and start the server """
         self.zenith_cli.run(['pg', 'create'])
+        self.config(config_lines)
         # FIXME: where did the name pg1 come from?
         self.zenith_cli.run(['pg', 'start', 'pg1'])
         self.running = True
 
+
+    #lines should be an array of valid postgresql.conf rows
+    def config(self, lines):
+        #TODO use real node name, not just guessed pg1
+        filename = 'pgdatadirs/pg1/postgresql.conf'
+        config_name = os.path.join(self.zenith_cli.env['ZENITH_REPO_DIR'], filename)
```

Comment:
It would probably be better to add `repo_dir` to `Postgres.__init__` and `postgres`, rather than peeking inside `ZenithCli.env`.

Reply:
It is a zenith repo_path, so I don't think it belongs to the Postgres structure, so I left it as is.
Though, I agree that it's a hack. It would be good to have pgdata_path in Postgres. I'll work on it.

Reply:
As long as this class needs access to that value, I ask that you retrieve this value from the fixture instead of peeking inside of another class's data.

Reply:
If we want the postgres data to live in another directory, we can calculate a new path, starting with the `test_output_dir` fixture.